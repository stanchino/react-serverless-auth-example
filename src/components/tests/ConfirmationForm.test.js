import React from "react";
import { ConfirmationForm } from "..";
import { testContainer } from "./shared-examples";

describe("PasswordResetConfirmForm", () => testContainer(<ConfirmationForm noEmail={null} />));