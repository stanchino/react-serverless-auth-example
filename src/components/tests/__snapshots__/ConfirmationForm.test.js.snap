// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordResetConfirmForm renders without errors 1`] = `
<form
  className={undefined}
  onSubmit={[Function]}
>
  <div
    className="messages"
  >
    <div
      className="flash"
    />
  </div>
  <div>
    <label />
    <div>
      <input
        name="code"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Confirmation Code"
        type="text"
        value=""
      />
    </div>
  </div>
  <button
    onClick={undefined}
    type="button"
  >
    Request New Code
  </button>
  <button
    disabled={true}
    type="submit"
  >
    Confirm
  </button>
  <button
    disabled={true}
    onClick={[Function]}
    type="reset"
  >
    Reset
  </button>
</form>
`;
